tests/Unit/Crypt/HashTest.php

```diff
@@ -183,6 +221,19 @@ public static function hmacData()
                 'The quick brown fox jumps over the lazy dog',
                 'e71aabb2588d789292fa6fef00b35cc269ec3ea912b1c1cd7127daf95f004a5df5392ee563d322bac7e19d9eab161932fe9c257d63e0d09eca0d91ab4010125e',
             ),
+            // from https://tools.ietf.org/rfc/rfc4231.txt
+            array(
+                'sha224',
+                str_repeat("\x0b", 20),
+                'Hi There',
+                '896fb1128abbdf196832107cd49df33f47b4b1169912ba4f53684b22',
+            ),
+            array(
+                'sha224',
+                'Jefe',
+                'what do ya want for nothing?',
+                'a30e01098bc6dbbf45690f3a7e9e6d0f8bbea2a39e6148008fd05e44',
+            ),
```

Comment:
Thanks for adding this! Just a few minor points more, in case it has been missed:

* I just noticed that the test vectors for HMAC-SHA-**384** are also missing here (they are also in https://tools.ietf.org/rfc/rfc4231.txt).
* There are more test cases for HMAC-SHA-256 and HMAC-SHA-512 (namely four each) than what has been added here - see, e.g., https://github.com/terrafrost/phpseclib/blob/089a4cb101c025611d066e13bd8c3e305c451558/tests/Unit/Crypt/HashTest.php#L160-L188.

Please also note the comment in https://github.com/phpseclib/phpseclib/pull/956#discussion_r58241646.

Thanks!